packages/test-helpers/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "dependencies": {
     "ethereumjs-abi": "^0.6.4",
-    "web3-eth-abi": "1.2.5",
-    "web3-utils": "1.2.5"
+    "web3-eth-abi": "^1.2.1",
+    "web3-utils": "^1.2.1"
```

Comment:
We can also move these up to be `^1.2.8`; does truffle pin them to `1.2.1` at the moment?

Reply:
Yes, truffle keeps it at `1.2.1`. Running `npm ls web3-utils` in the staking repo showed up to 5 different versions of it: `1.2.1`, `1.2.2`, `1.2.5`, `1.2.6` and `1.2.8` :see_no_evil:

Reply:
Haha 🤦‍♀️. How many of those can we control?